packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/TableDemo/TableActionsDemo.tsx

```diff
@@ -5,14 +5,20 @@ import {
   TableBody,
   TableProps,
   headerCol,
+  ICell,
   IRow,
   IRowData,
   IExtra,
-  IAction,
+  IActions,
```

Comment:
Would this be considered a breaking change? That is, the `IAction` interface has been renamed `IActions`

Reply:
I don't think so, it should work both ways. There are quite a few places in the table code where the type assertions are currently necessary so just trying to make the code more readable by favoring the "normal" approach of applying types where we can. If everything is read `as SomethingElse` it becomes a little hard to read. Does it make sense?

Reply:
Just want to make sure that someone isn't relying on the `IActions` interface and we break them by renaming it as `IAction`